aws-ecs/20-bastion.tf

```
@@ -13,33 +13,33 @@ data "aws_ami" "amazon_linux_2" {
 
 resource "aws_security_group" "bastion" {
   count  = var.create_bastion ? 1 : 0
-  name   = "bastion-sg"
+  name   = "${var.deployment_id}-bastion-sg"
   vpc_id = local.vpc_id
 
   egress {
-    description = "Allow All 443 Egress for SSM Accessye"
+    description = "Allow All 443 Egress for SSM Access"
     from_port   = 443
     to_port     = 443
     protocol    = "tcp"
     cidr_blocks = ["0.0.0.0/0"]
   }
 
   egress {
-    description = "Allow Internal 80 Egress pvault accss"
+    description = "Allow Internal 80 Egress pvault access"
```

```suggestion
    description = "Allow Internal 80 Egress PVault access"
```

Why do vault need port 80?